CONTRIBUTORS.rst

```diff
@@ -158,6 +158,7 @@ Listed in alphabetical order.
   Taylor Baldwin
   Théo Segonds             `@show0k`_
   Tim Freund               `@timfreund`_
+  Tim Godfrey              `@timmygee`_
```

Comment:
The second column is a reference to [a link in reStructuredText](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks), but it's not defined anywhere. You need to add it further down like the others.